homeassistant/components/light/zha.py

```diff
@@ -73,7 +73,34 @@
                         UNSUPPORTED_ATTRIBUTE):
                     discovery_info['color_capabilities'] |= \
                         CAPABILITIES_COLOR_TEMP
-        entities.append(Light(**discovery_info))
+
+        light_entity = Light(**discovery_info)
+
+        if discovery_info['new_join']:
+            from zigpy.zcl.clusters.general import OnOff, LevelControl
+            in_clusters = discovery_info['in_clusters']
+            if OnOff.cluster_id in in_clusters:
+                cluster = in_clusters[OnOff.cluster_id]
+                await helpers.configure_reporting(
+                    light_entity.entity_id,
+                    cluster,
+                    0,
+                    min_report=0,
+                    max_report=600,
+                    reportable_change=1
+                )
+            if LevelControl.cluster_id in in_clusters:
+                cluster = in_clusters[LevelControl.cluster_id]
+                await helpers.configure_reporting(
+                    light_entity.entity_id,
+                    cluster,
+                    0,
```

Comment:
For `level' reporting you want min_report interval of at least 1s or better 2-3s. GE dimmers would flood the zigbee network with attribute reports during on->off transitions.

Reply:
done